Dockerfile

```
@@ -24,6 +24,11 @@ RUN pip3 install --upgrade pip --no-cache-dir -r requirements.txt
 
 COPY ./app /Crafty_Controller_Fabric/app
 
+RUN mkdir -p /Crafty_Controller_Fabric/server/server_1
+
+COPY ./docker/minecraft_servers /Crafty_Controller_Fabric/server/server_1
```

minecraft_servers represents a folder that holds multiple servers i.e.

```
| minecraft_servers
\__ ServerA
\__ ServerB
\__ ServerC
\__ ServerD
```

So creating a directory labelled `server_1` isn't generalised enough. Let's do something more like this...


```suggestion
COPY ./docker/minecraft_servers /Crafty_Controller_Fabric/server
```

---

But if we added in multiple servers then they would all coexist within the same folder. Server 1 was a folder intended to individualise files so that we know which server they belong to